ChallengeTestRunner/challenge_test_lib/challenge_test.py

```diff
@@ -265,11 +265,15 @@ def challenge_fun(self, *args, **kwargs):
     def assert_printed(self, value, index=None):
         str_value = str(value)
         if index is not None:
-            return str_value in self.mock_print.printed[index]
+            self.assertTrue(str_value in self.mock_print.printed[index],
+                            msg=f'Value not found in print of index {index}')
```

Comment:
The server runs Python 3.5, which still didn't have f-strings. Please, use .format instead.

Reply:
Done